fix(docs) Resolve 196 mdx lint warnings for just, quickly, actually, PostgreSQL (#47358)
supabase/supabase: apps/docs/content/guides/auth/native-mobile-deep-linking.mdx

    - Go to your [auth settings](/dashboard/project/_/auth/url-configuration) page.
    - You need to enter your app redirect callback on `Additional Redirect URLs` field.

>     The redirect callback URL should have this format `[YOUR_SCHEME]://[YOUR_HOSTNAME]`. Here, `io.supabase.flutterquickstart://login-callback` is an example; you can choose whatever you would like for `YOUR_SCHEME` and `YOUR_HOSTNAME` as long as the scheme is unique across the user's device. For this reason, typically a reverse domain of your website is used.

Author: ```suggestion
    The redirect callback URL should have this format `[YOUR_SCHEME]://[YOUR_HOSTNAME]`. Here, `io.supabase.flutterquickstart://login-callback` is an example. You can choose whatever you would like for `YOUR_SCHEME` and `YOUR_HOSTNAME` as long as the scheme is unique across the user's device. For this reason, typically a reverse domain of your website is used.
```

    1. Go to your [auth settings](/dashboard/project/_/auth/url-configuration) page.
    2. Enter your app redirect URL in the `Additional Redirect URLs` field. This is the URL that the user gets redirected to after clicking a magic link.

>     The redirect callback URL must have the format [YOUR_SCHEME]://[YOUR_HOSTNAME]. For example: io.supabase.user-management://login-callback. You can use any values for YOUR_SCHEME and YOUR_HOSTNAME, but the scheme must be unique across the user's device. For this reason, a reverse domain of your website is typically used.

Reviewer: ```suggestion
    The redirect callback URL must have the format `[YOUR_SCHEME]://[YOUR_HOSTNAME]`. For example: `io.supabase.user-management://login-callback`. You can use any values for `YOUR_SCHEME` and `YOUR_HOSTNAME`, but the scheme must be unique across the user's device. For this reason, a reverse domain of your website is typically used.
```

Looks like your changes dropped some backticks here?

>     The redirect callback URL should have the format `[YOUR_SCHEME]://[YOUR_HOSTNAME]`. Here, `io.supabase.user-management://login-callback` is an example. You can choose whatever you would like for `YOUR_SCHEME` and `YOUR_HOSTNAME` as long as the scheme is unique across the user's device. For this reason, typically a reverse domain of your website is used.

Author: ```suggestion
    The redirect callback URL must have the format [YOUR_SCHEME]://[YOUR_HOSTNAME]. For example: io.supabase.user-management://login-callback. You can use any values for YOUR_SCHEME and YOUR_HOSTNAME, but the scheme must be unique across the user's device. For this reason, a reverse domain of your website is typically used.
```
